### DISCUSSION TITLE
Unity with Rider vs Unity with Consulo highlighting error

### DISCUSSION THREAD

**Discussion by feliperb98** [consulo/consulo#458] [Q&A] [answered]
is that a bad configuration by myself, or any kind of bug? because the highlighted error only happens with consulo :thinking: 

Rider
![image](https://user-images.githubusercontent.com/52469327/109004262-4b3ec700-7687-11eb-8d42-d6b23d27381c.png)

Consulo
![image](https://user-images.githubusercontent.com/52469327/109004714-e2a41a00-7687-11eb-8a3c-8bdb44f456e2.png)

PS: the error is highlighted, but I can run the game on unity editor without problems

**Comment by VISTALL** [comment, marked as answer]
Hello. Consulo have own C# analyzer (Rider use own). Due my limitations, my analyzer can provide sometimes bugs. They can be fxed - if you found issue pattern, and report to https://github.com/consulo/consulo-csharp

**Reply by feliperb98** [reply]
alright, thank you for the support, It's the first time it happened to me, if I discover the "pattern" I will report for you with pleasure :handshake:
